alpha/apps/kaltura/lib/clipconcat/kClipManager.php

```diff
@@ -282,17 +279,19 @@ private function handleClipConcatParentJob($batchJob)
 	 * @param int $priority
 	 * @return BatchJob[]
 	 * @throws APIException
-	 * @throws PropelException
+	 * @throws kCoreException
 	 */
 	private function addClipJobs($parentJob , $entryId, &$errDescription, $partnerId,
 	                             array $operationAttributes, $priority = 0)
 	{
 		$batchArray = array();
 		$order = 0;
+		$originalConversionEnginesExtraParams =
+			assetParamsPeer::retrieveByPK($operationAttributes[0]->getAssetParamsId())->getConversionEnginesExtraParams();
```

Comment:
Use const SYSTEM_DEFAULT_FLAVOR_PARAMS_ID instead  of $operationAttributes[0]->getAssetParamsId()